packages/webapp/src/containers/Filter/Tasks/index.jsx

```diff
@@ -166,6 +166,7 @@ const TasksFilterPage = ({ onGoBack }) => {
     },
     {
       subject: t('TASK.FILTER.DATE_RANGE'),
+      filterKey: FilterType.DATE_RANGE,
```

Comment:
To keep it consistent with others should we get this from the constants file? The Filter types I could see being improved in the future.

Reply:
That's an option I considered too, but I wasn't sure if creating a new constant just for this was an overkill. I will create it if I work in a new ticket that requires that same value 👍